solrmonitor/solrmonitor.go

```diff
@@ -275,12 +275,9 @@ func (c *SolrMonitor) updateCollectionState(path string, children []string) (map
 	for _, shard := range coll.collectionState.Shards {
 		for rname, rstate := range shard.Replicas {
 			if prs, found := rmap[rname]; found {
-				//if version is more than current then take that state
-				if prs.Version > rstate.Version {
-					rstate.Version = prs.Version
-					rstate.Leader = prs.Leader
-					rstate.State = prs.State
-				}
+				rstate.Version = prs.Version
+				rstate.Leader = prs.Leader
+				rstate.State = prs.State
```

Comment:
We don't need this check as we should take the latest as is represented in Zookeeper.